src/site/antora/modules/ROOT/pages/graalvm.adoc

```diff
@@ -0,0 +1,111 @@
+////
+ Licensed to the Apache Software Foundation (ASF) under one or more
+ contributor license agreements. See the NOTICE file distributed with
+ this work for additional information regarding copyright ownership.
+ The ASF licenses this file to You under the Apache License, Version 2.0
+ (the "License"); you may not use this file except in compliance with
+ the License. You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+////
+
+= Building GraalVM native images
+
+Since version
+xref:release-notes.adoc#release-notes-2-25-0[`2.25.0`]
+both
+xref:manual/api.adoc[Log4j API]
+and
+xref:manual/implementation.adoc[its reference implementation]
+provide out-of-the-box support for creating native executables using
+https://www.graalvm.org/[GraalVM].
+
+Depending on the logging implementation you are using, additional steps might be required to compile a Log4j API application with GraalVM.
```

Comment:
I am confused by the introduction and the sectioning following it. What is this page exactly documenting? Guiding applications using Log4j API to create GraalVM native images? What about applications using Log4j Core in combination with SLF4J?

Could you end the introduction paragraph with a dumb-proof _"This document guides you on ..."_ explanation, please?

Reply:
I am documenting the usage of GraalVM with applications written against the Log4j API.
Honestly I didn't test the usage of third-party logging APIs, although I don't believe our bridges use any reflection, so they should be OK.

I modified the introduction in:
https://github.com/apache/logging-log4j2/pull/3054/commits/5031b1bd30b78271d3607b7456d22258376becdb